APE_specutil.rst

```diff
@@ -0,0 +1,482 @@
+
+* `specviz`. This package will provide 1D spectrum visualization capabilities,
+  but is scoped to be focused primarily on that, and not arbitrarily complex
+  spectrum visualization (e.g., data cubes). Note that, unlike the above two,
+  this package should *not* be considered a toolbox that other tools should be
+  derived from.  While other packages are welcome to use it, it is meant to
+  provide a critical piece of functionality, while acknowledging that
+  visualization, GUI code, and even personal taste are variable enough that
+  there is room for a few other visualization packages in the ecosystem. It
+  will, however, have a plugin architecture that may allow users/developers to
+  add to it.
+
+In addition to the packages described above, there are a wide range of more
+domain-specific, experimental, or taste-specific packages that are likely to be
+desired in the field of astronomical spectroscopy. These packages can be
+developed as independent affiliated packages, and this APE encourages such
+development.  However, the intent is for such packages to not *duplicate*
+behavior in the foundational three packages described above. Because of the
+wide range of scientific use cases for spectra, it is understood that some
+level of duplication between affiliated packages is expected.  E.g., multiple
+line-fitting packages may be reasonable because different science cases may be
+better-suited to different line-fitting approaches.
+
+Packages should aim for the same level of testing and documentation found in
+the `astropy` package, and should follow the standard Astropy affiliated
+package process to do this.  This includes, in particular, considering overlap
+with existing packages - existing code should be considered and merged when
+appropriate.  The Spectroscopic Coordinating Committee will assist the Astropy
+Coordinating Committee with this process.  Developers are encouraged to build
+or update existing tools to meet their needs rather than create new packages.
+
+The Spectrum1D class
+--------------------
+
+A spectroscopy package should provide a representation of the data, tools for
+reducing spectroscopic observations, analysis tools, and methods for
+interacting and visualization of the spectra. In this section, we describe some
+of the functionality that will be required for a common spectroscopic package.
+While equally important, we leave the specification of multi-dimensional
+spectral objects with different spectral axes to future work.  At the same
+time, we should consider how these tools will be compatible with
+multi-dimensional data in the future.
+
+
+Attributes of Spectrum1D
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this context, a spectrum describes how the flux changes as a function of the
+energy of the radiation from some object.  There are a number of other
+parameters that can be used to describe a spectrum, and a ``Spectrum1D`` object
+will have the following properties:
+
+* ``spectral_axis``
+* ``flux``
```

Comment:
Apart from the obvious physical naming, what was the logic behind going for 'flux' and not 'data' here?

We have adopted NDData for a lot of multi-dimensional spectral (and not spectral) work around SunPy so far [1], I would like to see some level of API consistency between `Spectrum1D` and `NDData`, which there appears to be (i.e. uncertainty, meta).

What would people think of aliasing `.data` to `.flux`? 

[1] We did a lot of stuff on 2D+ spectral data (slit spectrograph) this summer see these repos if you are curious: https://github.com/sunpy/ndcube/ https://github.com/sunpy/irispy

Reply:
Thanks for the feedback @Cadair -- yes, `Spectrum1D` will inherit from `NDData` so `.data` and `.flux` will be aliased.   I'll add that to the APE.